include/core.h

```diff
@@ -254,7 +254,11 @@ extern u16 gBgCntRegs[4];
 //       [4]{s16 x, s16 y}
 //       Should we introduce a
 //       "#define NUM_BACKGROUNDS 4" in gba/defines.h?
+#if PLATFORM_GBA
```

Comment:
I could convert this to `WIDESCREEN_HACK`? Though I don't see an issue with these being bigger